@carbon/icons-react.js

```diff
@@ -0,0 +1,10 @@
+if (!window.carbonicons) {
+  throw new Error('Not compatible: @carbon/icons-react is not available in this environment.');
```

Comment:
Let's clearly identify what version of the environment is expected, i.e. a particular Desktop Modeler version:

```suggestion
  throw new Error('Dependency @carbon/icons-react is only available in Desktop Modeler v5.33.0 and newer.');
  
```
  throw new Error('Not compatible: @carbon/icons-react is not available in this environment.');
```